### ISSUE TITLE
Don't backup 'searchable content'

### ISSUE THREAD

**Issue by epheterson** [msolters/Tiles#103]
This should be generated and stored on the internal db only, there's no benefit that I can tell to have the user backup and restore this information.

**Comment by msolters** [comment]
No, this should definitely be backed up.  Searchable content is not a backend process, believe it or not.  It's generated by the browser.  I could use regex to strip out the HTML but it's very, very expensive for a server to do REGEX (in terms of CPU) and it always has a failure rate.

The only thing that is near-perfect at getting the human readable text from a chunk of HTML is, I'm sure you'll appreciate this logic -- a web browser.  It's like...what it's made for.

So in other words getting searchable content is a totally client-side thing, not server-side.

**Comment by epheterson** [comment]
Couldn't it re-render it on import? This will essentially double the side of backups!

**Comment by msolters** [comment]
No, it will double database write-time.  Write to DB is the single most expensive thing to do with a server.  The increase in size is trivial compared to that throughput cost.

**Comment by epheterson** [comment]
Okay, you're the magician
